engine/src/main/java/nl/inl/blacklab/search/results/ResultsStats.java

```diff
@@ -5,6 +5,8 @@
 
 public abstract class ResultsStats implements SearchResult {
 
+    public static final ResultsStats SEARCH_NOT_STARTED_YET = new ResultsStatsStatic(0, 0, MaxStats.NOT_EXCEEDED);
```

Comment:
I think a better place for this constant is in the `ResultStatsStatic` class itself. It might be also [safer](http://ternarysearch.blogspot.com/2013/07/static-initialization-deadlock.html) [to do so](https://bugs.openjdk.java.net/browse/JDK-8037567)

Reply:
Good tip. In the meantime however, I realized I can't use ResultsStatsStatic here because I need done() to return false. Also, processedTotal() and countedTotal() should never be called on this object. I created a custom implementation that shouldn't have the risk of deadlock either.

(the deadlock issue is unfortunate in general however, as I would dislike to tie constants like this to an implementation. I suppose a factory method would solve this though)